hivemq-edge/src/frontend/src/extensions/datahub/components/nodes/TransitionNode.spec.cy.tsx

```diff
@@ -0,0 +1,53 @@
+/// <reference types="cypress" />
+
+import { NodeProps } from 'reactflow'
+
+import { mockReactFlow } from '@/__test-utils__/react-flow/providers.tsx'
+import { MOCK_DEFAULT_NODE } from '@/__test-utils__/react-flow/nodes.ts'
+
+import { DataHubNodeType, TransitionData } from '../../types.ts'
+import { TransitionNode } from './TransitionNode.tsx'
+
+const MOCK_NODE_TRANSITION: NodeProps<TransitionData> = {
+  id: 'node-id',
+  type: DataHubNodeType.TRANSITION,
+  data: {},
+  ...MOCK_DEFAULT_NODE,
+}
+
+describe('TransitionNode', () => {
+  beforeEach(() => {
+    cy.viewport(400, 400)
+  })
+
+  it('should render properly', () => {
+    cy.mountWithProviders(mockReactFlow(<TransitionNode {...MOCK_NODE_TRANSITION} selected={true} />))
+    cy.getByTestId(`node-title`).should('contain.text', 'Transition')
+    cy.getByTestId(`node-model`).should('contain.text', '< not set >')
+
+    // TODO[NVL] Create a PageObject and generalise the selectors
```

Comment:
question: What is the process on TODOs? Is that being added to the backlog

Reply:
> question: What is the process on TODOs? Is that being added to the backlog

If they come with a ticket number (`TODO[18458]`), they are in the backlog and will be removed when solved.
The others should be added to the backlog and processed... at some point :-)
I usually do that when I come across one that I am in a position to address.